discovery_engine/lib/discovery_engine.dart

```diff
@@ -19,4 +19,6 @@ library discovery_engine;
 
 export 'package:xayn_discovery_engine/src/api/api.dart';
 export 'package:xayn_discovery_engine/src/discovery_engine_base.dart';
+//FIXME: remove once unneded
```

Comment:
```suggestion
//FIXME: remove once unneeded
```
When it will be unneeded?

Reply:
When we have any functionality which actually uses async bindgen;